src/main/java/org/jitsi/jicofo/Bridge.java

```diff
@@ -122,7 +95,7 @@ private static Integer getInt(Object obj)
     /**
      * The last known {@link ColibriStatsExtension} reported by this bridge.
      */
-    private ColibriStatsExtension stats = null;
+    private ColibriStatsExtension stats = new ColibriStatsExtension();
```

Comment:
maybe a bit nit-picky, but should we define a single static instance to use for instantiation here?

i.e.
```
private static final EMPTY_STATS_EXTENSION = new ColibriStatsExtension();
...
private ColibriStatsExtension stats = EMPTY_STATS_EXTENSION;
```

i like being able to eliminate the null checks btw!